app/scripts/modules/google/src/serverGroup/configure/wizard/zones/zoneSelector.directive.html

```diff
@@ -22,4 +22,14 @@
       </li>
     </ul>
   </div>
+  <div class="col-md-7" ng-if="vm.command.selectZones">
+    <ui-select multiple
```

Comment:
Should have a minimum of 1 zone selected I think - from the Clouddriver PR, if you select 0 zones, I think you'll end up with the default 3.

Reply:
Agreed, added validation on this.